src/js/model/validation.js

```diff
@@ -0,0 +1,14 @@
+import {CAR} from "../constants/constants.js";
+
+export const validateCarNames = (namesArr) => {
+  const namesError = namesArr
+    .split(",")
+    .filter((name) => name.trim().length >= CAR.NAME_MAX_LENGTH).length;
```

Comment:
nameErrors로 배열로 받는것보다 에러가 1개라도 있는지 검사하는게 목적이니 이렇게 작성해보면 어떨까요?
```suggestion
  const isCarValidationValid = namesArr
    .split(",")
    .filter((name) => name.trim().length >= CAR.NAME_MAX_LENGTH).length === 0;
  
  return isCarValidationValid;
```